SWLOR.Game.Server/Service/TaxiService/TaxiDestinationType.cs

```diff
@@ -29,6 +29,8 @@ public enum TaxiDestinationType
         VelesFoszEstate = 9,
         [TaxiDestination(1, "Veles Apartments", "TAXI_VELES_APARTMENTS", 100)]
         VelesApartments = 10,
+        [TaxiDestination(1, "Outpost Hope", "TAXI_VELES_OUTPOST", 1000)]
+        OutpostHope = 11,
```

Comment:
I don't want factions on the Taxi system. If they want quick travel, they can start up a city. Please remove.